autonomy/chain/mint.py

```diff
@@ -185,6 +188,7 @@ def mint_service(  # pylint: disable=too-many-arguments
     threshold: int,
     owner: Optional[str] = None,
     timeout: Optional[float] = None,
+    l2: bool = False,
```

Comment:
Don't make this a flag. The rule is: Ethereum -> standard registry, all other networks -> l2 registry. No need for user having to know this.